sqlitecluster/SQLiteNode.cpp

```diff
@@ -2405,13 +2286,11 @@ void SQLiteNode::_handlePrepareTransaction(SQLite& db, SQLitePeer* peer, const S
     }
     uint64_t leaderSentTimestamp = message.calcU64("leaderSendTime");
     uint64_t transitTimeUS = dequeueTime - leaderSentTimestamp;
-    uint64_t threadStartTimeUS = threadStartTime - dequeueTime;
-    uint64_t applyTimeUS = STimeNow() - threadStartTime;
+    uint64_t applyTimeUS = STimeNow() - prepareStartTime;
     float transitTimeMS = (float)transitTimeUS / 1000.0;
-    float threadStartTimeMS = (float)threadStartTimeUS / 1000.0;
     float applyTimeMS = (float)applyTimeUS / 1000.0;
     PINFO("[performance] Replicated transaction " << message.calcU64("NewCount") << ", sent by leader at " << leaderSentTimestamp
-          << ", transit/dequeue time: " << transitTimeMS << "ms, thread start time: " << threadStartTimeMS << "ms, applied in: " << applyTimeMS << "ms, should COMMIT next.");
+          << ", transit/dequeue time: " << transitTimeMS << "ms, applied in: " << applyTimeMS << "ms, should COMMIT next.");
```

Comment:
we need to fix https://github.com/Expensify/Salt/blob/f1e3be3327597d36820db673ae589ebdaf4a6878/shared_pkgs/bedrock/files/logsToStatsd.pl#L232-L242 too

Reply:
Done here: https://github.com/Expensify/Salt/pull/14370